src/index.ts

```diff
@@ -226,3 +226,18 @@ export { acp_v1 } from "./acp/v1";
  * them will be included in your bundle.
  */
 export { acp_v2 } from "./acp/v2";
+
+/**
+ * This API is still experimental, and subject to change. It builds on top of both
+ * ACP and ACL, aiming at being adaptable to any Access Control system that may be
+ * implemented in Solid. That is why it is purely Resource-centric: the library
+ * discovers metadata associated to the Resource itself, and calls the appropriate
+ * underlying API to deal with the Access Control in place for the target Resource.
+ *
+ * Being still developped, the following export is *only* intended for experimentation
```

Comment:
In the case of this API, we're only including this object because not all bundlers support exports maps yet. So it's probably a good idea to add a note here that we recommend importing directly from `@inrupt/solid-client/access/universal` if you're using Node 12+, Webpack 5+, or anything else that supports exports maps.

Reply:
2a7b6b9c2ebc2d1e971724c29cb4989774f74224